cms/djangoapps/contentstore/views/item.py

```diff
@@ -479,6 +489,14 @@ def _create_item(request):
         if display_name is not None:
             metadata['display_name'] = display_name
 
+        # Entrance Exams: Chapter module positioning
+        child_position = None
+        if settings.FEATURES.get('ENTRANCE_EXAMS', False):
+            is_entrance_exam = request.json.get('is_entrance_exam')
+            if category == 'chapter' and is_entrance_exam:
+                metadata['is_entrance_exam'] = is_entrance_exam
+                child_position = 0
+
```

Comment:
Hmm...there really isn't a way to introduce that scenario at the moment because it's all in the system.  If someone did end up modifying the system to support alternative module types being flagged as entrance exams then the tests would need to be modified accordingly to support the changes.  And if anything I think in that scenario we'd *remove* the chapter constraint.  So I guess it doesn't seem necessary to me -- thoughts?